ubuntu/debian/libignition-gazebo.install

```diff
@@ -2,3 +2,5 @@ usr/lib/*/*.so.*
 usr/share/ignition/ignition-gazebo[0-99]/*.config
 usr/share/ignition/ignition-gazebo[0-99]/gui/*
 usr/share/ignition/ignition-gazebo[0-99]/worlds/*
+tools/desktop/ignition-gazebo.svg usr/share/icons/hicolor/scalable/apps/
+tools/desktop/ignition-gazebo.desktop usr/share/applications/
```

Comment:
Can we install these in versioned files, so we can support Citadel and Fortress side-by-side?

Reply:
oh good catch. Renaming is not as trivial as adding the new name here. Instead of that, I've modified upstream code to produce a versioned file directly in [7e8d017](https://github.com/ignitionrobotics/ign-gazebo/pull/1411/commits/7e8d017a1bfaecd84726a173d3b27fec971e14c9)